webextensions/api/permissions.json

```
@@ -174,7 +174,7 @@
                 ]
               },
               "firefox_android": {
-                "version_added": true
+                "version_added": false
```

The method is available and was supported until version 68.

The only issue is that there is no UI with a prompt. If an optional permission has already been granted, then there won't be a prompt. So the API can still be marked as supported, and you should use:
```suggestion
                "version_added": true,
                "partial_implementation": true,
                "notes": "Since version 79, the UI to approve the permission request is missing (<a href='https://bugzil.la/1601420'>bug 1601420</a>)."
```

---

I feel like the entire point of `permissions.request` is so the user is prompted if the permission if not already given, so saying the API is partially implemented as `true` is pretty misleading. Could we have `"version_added": false` and still have the partial implementation asterisk and note?

---

`version_added: false` implies that it was never supported. If any, `version_removed` should be added.

From the extension's point of view, the API is supported. It validates parameters and may sometimes even return something. There have been bugs where the API did not work as desired (see e.g. https://bugzilla.mozilla.org/show_bug.cgi?id=1433292#c14). In these cases, the API itself is still supported, but buggy. For that, the `notes` field fits perfectly.

---

Thanks for the clarification!